README.md

```diff
@@ -1,76 +1,124 @@
 ---
 services: active-directory
-platforms: java
+platforms: Java
 author: xerners
+level: 200
+client: Java Console Application
+service: Microsoft Graph
+endpoint: AAD V1
 ---
+# Integrating Azure AD into a Java console application using username and password
 
-# Integrating Azure AD into a Java command line using username and password
+## About this sample
 
-This sample demonstrates a command line application calling a web API that is secured using Azure AD. The Java application uses the Active Directory Authentication Library for Java (ADAL4J) to obtain a JWT access token through the OAuth 2.0 protocol. The access token is sent to the web API to authenticate the user. This sample shows you how to use ADAL to authenticate users via raw credentials (username and password) via a text-only interface.
+### Overview
 
+This sample demonstrates a Java console application calling Microsoft Graph that is secured using Azure Active Directory.
+
+1. The Java application uses the Active Directory Authentication Library for Java (ADAL4J) to obtain a JWT access token through the OAuth 2.0 protocol.
+2. The access token is used as a bearer token to authenticate the user when calling the Microsoft Graph.
+
+### Scenario: 
+
```

Comment:
you might want to add a picture with the topology (See @abhidnya13 who has done it for one of her's)

Reply:
done